enable.ps1

```diff
@@ -1,48 +1,41 @@
-###########################################
+#################################################
 # HelloID-Conn-Prov-Target-Inception-Enable
-#
-# Version: 1.0.1
-###########################################
-# Initialize default values
-$config = $configuration | ConvertFrom-Json
-$p = $person | ConvertFrom-Json
-$aRef = $AccountReference | ConvertFrom-Json
-$success = $false
-$auditLogs = [System.Collections.Generic.List[PSCustomObject]]::new()
-
+# PowerShell V2
+# Version: 1.0.0
+#################################################
+Write-Verbose -Verbose $actionContext.References.Account
 # Enable TLS1.2
 [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor [System.Net.SecurityProtocolType]::Tls12
 
 # Set debug logging
-switch ($($config.IsDebug)) {
+switch ($($actionContext.Configuration.isDebug)) {
     $true { $VerbosePreference = 'Continue' }
     $false { $VerbosePreference = 'SilentlyContinue' }
 }
 
-function Get-RandomCharacters($length, $characters) { 
-    $random = 1..$length | ForEach-Object { Get-Random -Maximum $characters.length }
-    return [String]$characters[$random]
-}
+# Script Variables
+#$employeeFound = $false
+#$userFound     = $false
```

Comment:
$employeeFound and $userFound can be removed? (use ctrl F to check code)

Reply:
Removed